### PR TITLE
Add lint to CI

### PR DESCRIPTION
**Pull request by alexander-demichev**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,6 +27,8 @@ ci: build test
 test: ## Test the project
 	GO111MODULE=on go test ./...
 
+ci-lint: lint fmt vet
+
```
Comment (jmrodri) [inline Makefile]: Any reason not to add this to the ci target? 

```
ci: lint fmt vet build test
```

Devs can just run them individually locally, most often `make lint` is run.

File: .travis.yml [hunk 2]
```diff
@@ -18,5 +18,10 @@ matrix:
         directories:
           - $HOME/Library/Caches/go-build
           - $HOME/gopath/pkg/mod
+
+before_install:
+  - env GO111MODULE=off go get -u golang.org/x/lint/golint
+
 script:
+  - make ci-lint
```
Comment (jmrodri) [inline .travis.yml]: +1 to adding this to travis, see my other comment about possibly combining the 2 targets.

Comment (eriknelson) [reply .travis.yml]: Are you upvoting splitting the linting out from the build/test? Or just the general idea we should add this stuff to travis?

**Comment by alexander-demichev** [comment]
@eriknelson PR is ready for another review

**Pull request merged**